src/lib/viewers/doc/DocControls.tsx

```diff
@@ -40,12 +40,13 @@ export default function DocControls({
     pageCount,
     pageNumber,
     scale,
+    isThumbnailsOpen,
```

Comment:
Since we still need to add tests, can we alphabetize the props here and in ThumbnailsToggle, while we're at it?

Reply:
yes! Totally